src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -21,13 +21,14 @@ public void registerWith( final CmdGeneral cmdLine ) {
 	}
 
 	@Override
-	public void processArgs( final CmdArgModule cmdLine ) {
+    public void processArgs(final CmdArgModule cmdLine) {
```

Comment:
Please revert the change to this line. You can do so directly by applying the edit suggestion below.

Two things about this:
1. Please don't change from tab-based indentation to space-based indentation. In particular not if you do it in only one line without changing it anywhere else in the file because that makes the file look like a mess if I view it with a different tab size than you (but also don't do it in general--I prefer tab-based).
2. Please don't remove the spaces inside the parenthesis if there are such spaces everywhere else in the file. That makes it look a bit messy because of the inconsistency across the file. If you start a new file and prefer no such spaces there, then that's fine--as long as you keep it consistent within the file.

```suggestion
	public void processArgs( final CmdArgModule cmdLine ) {
```